driver-embedded/build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
 
     testCompile project(':bson').sourceSets.test.output
     testCompile project(':driver-core').sourceSets.test.output
+    testCompile project(':driver-sync').sourceSets.test.output
```

Comment:
;) No copied JsonPowerCrudTestHelper for embedded

Reply:
That's a relief!